xcodeproj/internal/bazel_integration_files/xcodeproj.bazelrc

```diff
@@ -34,6 +34,16 @@ build:rules_xcodeproj --features=relative_ast_path
 # this in `swift_debug_settings.py`, so debugging still works.
 build:rules_xcodeproj --features=swift.cacheable_swiftmodules
 
+# `swift.index-while-building` passes `-index-store-path` to `swiftc`, which
+# generates index store data. We enable this globally, even though we only need
+# it for the normal build, since it's better to get cache hits between normal
+# and Index Build builds than it is to save some time in an uncached Index Store
+# build. It also makes it easier to disable in your `.bazelrc` files. We only
+# download the outputs in the normal build though, saving some bandwidth/disk
+# space.
+build:rules_xcodeproj --features=swift.index_while_building
+build:rules_xcodeproj --features=swift.use_global_index_store
```

Comment:
Possibly, yes. Though Xcode does this, and developers are used to the fast bootstrapping of indexing data. But it will be 100% needed for focused projects.